deltachat-ffi/deltachat.h

```diff
@@ -441,7 +441,24 @@ char*           dc_get_info                  (dc_context_t* context);
  */
 char*           dc_get_oauth2_url            (dc_context_t* context, const char* addr, const char* redirect_uri);
 
+/**
+ * Get the provider information from our provider overview as json object.
+ */
+char*           dc_get_json_provider_info_from_email  (char* email);
+
+typedef struct _dc_provider  dc_provider;
+
+dc_provider*    dc_get_provider_info_from_email       (char* email);
+char*           dc_provider_get_overview_page         (const dc_provider* provider);
+char*           dc_provider_get_name                  (const dc_provider* provider);
+char*           dc_provider_get_markdown              (const dc_provider* provider);
+char*           dc_provider_get_status_date           (const dc_provider* provider);
+int             dc_provider_get_status                (const dc_provider* provider);
+
 
+#define         DC_PROVIDER_STATUS_OK           1
+#define         DC_PROVIDER_STATUS_PREPARATION  2
+#define         DC_PROVIDER_STATUS_BROKEN       3
```

Comment:
All functions and constants in this file have pretty comprehensive docs currently, would it be possible to keep that so for new ones?

Reply:
Done.

Reply:
thanks, asked for some more detail.